HPS/Heartland/Block/Orderreview.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ *  Heartland payment method model
+ *
+ * @category    HPS
+ * @package     HPS_Heartland
+ * @author      Heartland Developer Portal <EntApp_DevPortal@e-hps.com>
+ * @copyright   Heartland (http://heartland.us)
+ * @license     https://github.com/hps/heartland-magento2-extension/blob/master/LICENSE.md
+ */
+namespace HPS\Heartland\Block;
+ 
+class Orderreview extends \Magento\Framework\View\Element\Template
+{
+    public function getOrderId()
+    {       
+		return $_GET['oid'];
```

Comment:
format needs to be fixed in all the files

Reply:
Format has been fixed in all the files